src/styles/_star.scss

```diff
@@ -0,0 +1,21 @@
+@use './variables';
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+
+  &-star {
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    background: no-repeat url(/src/images/star-active.svg);
```

Comment:
Ensure that the star image path is correct and that the image exists at the specified location. The path should be relative to the project structure.